iphone/Classes/TiLogServer.m

```diff
@@ -223,6 +242,23 @@ +(void)startServer
 		connections = [[NSMutableArray alloc] init];
 	}
 
+	if (headers == nil) {
+		NSString* version = [NSString stringWithCString:TI_VERSION_STR encoding:NSUTF8StringEncoding];
+		NSDictionary* map = [[NSDictionary alloc] initWithObjectsAndKeys:
+			TI_APPLICATION_NAME, @"name",
+			TI_APPLICATION_ID, @"appId",
+			TI_APPLICATION_VERSION, @"version",
+			TI_APPLICATION_DEPLOYTYPE, @"deployType",
+			TI_APPLICATION_GUID, @"guid",
+			version, @"tiSDKVersion",
+			@"__GITHASH__", @"githash",
+			nil];
+
+		headers = [[[[TiUtils jsonStringify:map] stringByAppendingString:@"\r\n"] dataUsingEncoding:NSUTF8StringEncoding] retain];
```

Comment:
i wonder if this `headers` has to be released eventually somewhere. @hansemannn thoughts?